modules/release-notes/pages/sdk-release-notes.adoc

```diff
@@ -35,60 +35,59 @@ This {release} version of the software and programming language should not be us
-- The name of the built-in {proglang} library has changed from `stdlib` to `base`.
-- All of the module names in the `base` library start with an upper case letter.
-- The `principalId` module has been changed to `Principal`.
+- You can now look up a canister identifier using the command `dfx canister id <canister_name>`.
+- The `--check` flag can be used with the `dfx build` command to check whether a canister will build before creating or building the canister.
+- Both canister name and identifiers are printed upon canister creation.
+- Canister manifest now stores multiple canister IDs associated with local and remote networks as well as network mapping.
 
-=== Modifying existing programs
+=== Tungsten (Developer Network)
 
-If you have programs that rely on modules in the `stdlib` library, you must change the `import` statement to use the new `base` library name.
-For example, you might have previously imported the Array module using the following statement:
+- HTTP authorization and credentials management for onboarded Tungsten users.
+- `Tungsten` added as a provider in `dfx.json`
+- You can use the `dfx ping` command to access and toggle between local and Tungsten networks.
+- The `--network <network>` flag can be used to build and install canisters to the specified provider.
+- Canister ID formatting for accessing Tungsten-deployed apps in the browser.
 
-[source,motoko]
-----
-import Array "mo:stdlib/array";
-----
+=== Motoko
 
-With this release, you would change the statement like this:
+- Stable variable support
+- Improved type checking of literals as payload for variant tags. The following will now type-check:
+`func () : { #tag : Nat8 } = #tag 42;`
+- Warning when the read file has a different name from the requested file.
+- `debug_show` implemented for `Principal`
+- Type hash implementation
 
-[source,motoko]
-----
-import Array "mo:base/Array";
-----
-=== Addressing case-sensitivity
+== Breaking changes
+The {release} release includes the following changes that might require updates to existing programs:
 
-If your local development environment runs on macOS, you might be able to build your {proglang} program using a lower case module name because the macOS file system is not case-sensitive by default. 
- 
-If you use lower case module names, however, the same code will not compile on Linux computers. 
-Therefore, the recommended best practice is to always capitalize module names.
-For example, use `import Array "mo:base/Array"`, not `import Array "mo:base/array"`.
+- The command `dfx new` now creates a separate assets canister. Programs built with earlier versions of the SDK may need to be converted to this new format. 
+- You must now create empty canisters before building and installing using the subcommand `dfx canister create`.
+- `string` and `number` are no longer supported as arguments to the `--type` flag for `dfx canister call` commands.
+- Methods starting with `__` will break, as those are not implemented anymore.
```

Comment:
Where does this come from? I'm not aware of this change.

Reply:
@chenyan-dfinity I dug it out of this PR, any ideas? https://github.com/dfinity-lab/sdk/pull/804

Reply:
I see. These are very internal changes. It doesn't affect external developers.

Reply:
Removed.